chrome_manifest_v3/service_worker.js

```
@@ -1,79 +1,79 @@
+    chrome.storage.sync.get('debugMode', function (response) {
+      if (response.hasOwnProperty('debugMode') && response['debugMode']) {
+        debugMode_g = true;
+      }
+    });
+  }
 
-    if (debugMode_g) {
-        console.log("g2t→" + data);
-    }
+  if (debugMode_g) {
+    console.log('g2t→' + data);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)